### ISSUE TITLE
Split languages into separate files 

### ISSUE THREAD

**Issue by offline-first** [Tienisto/flutter-fast-i18n#49]
My project has over 7k source keys and over 14 languages. This makes the generated file over 2.5 MB in size, thus the IDE has problems caching/indexing the file. Maybe it is better to split the classes into separate files?

**Comment by Tienisto** [comment]
Hi,

I can add a new option to split into multiple files. Like `mode: split`

```text
strings.g.dart - base translations and base functionality
strings_fr.g.dart
strings_de_DE.g.dart
strings_map.g.dart - the flat map containing all translations
```

**Comment by offline-first** [comment]
@Tienisto 
This looks good, I think ARB also stores the files separately. However I could now solve the problem by increasing the cache in the IDE.

Maybe you should think again if it is the best solution to store the maps in dart files since they are all loaded into memory.
Probably a better solution in large projects is to load the json files raw from the assets folder?

What do you think?

**Comment by Tienisto** [comment]
Yep, I know that the map can be quite large. That's why the map is declared as a lazy late variable and should be only initialized on first call.

**Comment by Tienisto** [comment]
I just released `output_format` in `5.7.0`.
You can set `output_format: multiple_files` to try this feature out. I am curious if it helped solving your issue :)

**Comment by offline-first** [comment]
Thank you very much, that solved my problem!
